res/values/strings.xml

```diff
@@ -311,6 +311,7 @@ there are any).</string>
     <string name="np_desc_it_networks" translatable="false">ATM, GTT, AMAT, ACTV</string>
 
     <string name="np_region_liechtenstein">Liechtenstein</string>
+    <string name="np_name_vmobil">VVV</string>
```

Comment:
No, I didn't test it. I suppose it's just renaming the "VAO" to "VVV", similar to what happens with db here:
```
    <string name="np_name_db" translatable="false">Deutsche Bahn</string>
    <string name="np_desc_db">Most of Germany</string>
```